From 4f4c2e2ef3db99b0a7dfbb4e116d4cd8a75e8eba Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Wed, 11 Sep 2019 15:20:46 +0800
Subject: [PATCH 105/245] arm64: dts: ls1028a: Correction the core clock of the
 HDP

commit 7772bf570b4a890c816f48acabd72912b31319cf from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

This clock should be use /4 of the ACLK frequency, so choosed
the clock provider's CGA_PLL2/2.

Signed-off-by: Wen He <wen.he_1@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 32b96df82bb1..16e52a456dac 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -797,7 +797,7 @@
 		compatible = "fsl,ls1028a-dp";
 		reg = <0x0 0xf200000 0x0 0xfffff>;
 		interrupts = <0 221 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&sysclk>, <&hdpclk>, <&hdpclk>,
+		clocks = <&sysclk>, <&clockgen 2 2>, <&hdpclk>,
 			 <&hdpclk>, <&hdpclk>, <&hdpclk>, <&hdpclk>;
 		clock-names = "clk_ipg", "clk_core", "clk_pxl",
 			      "clk_pxl_mux", "clk_pxl_link", "clk_apb",
-- 
2.17.1

